tmpl/wiki/tutorial-lwt.md

```diff
@@ -74,508 +111,327 @@ Assuming you are building for POSIX using native kernel sockets, compile the app
 ```
 
 If you are building for a different MirageOS platform, change the `--unix` switches appropriately (presently either `--xen` or `--unix`).
```

Comment:
The "appropriately-presently" is a bit nebulous for someone who doesn't know much about MirageOS, maybe just say if you are building for Xen, change `--unix` to `--xen`?